meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb

```diff
@@ -7,10 +7,16 @@ inherit packagegroup
 
 PACKAGES = " \
   ${PN}-system \
+  ${PN}-webkit \
 "
 
 RDEPENDS:${PN}-system = " \
   packagegroup-core-base-utils \
   chrony \
   chronyc \
 "
+
+RDEPENDS:${PN}-webkit = " \
```

Comment:
@WiktorG351 You should add this to `IMAGE_INSTALL` as well, e.g.:

```bb
IMAGE_INSTALL:append = " \
  packagegroup-system \
  packagegroup-security \
"
```

Also consider renaming the packagegoup to smth like `${PN}-graphics` or `${PN} - multimedia`.